docs/user/unofficial-projects.rst

```diff
@@ -1,96 +1,132 @@
+* Social media accounts associated with the project
+* The project's primary repository -- README files, repository description, or source code
 
-* It is not hosted on an official domain of the project
-* It is not linked from the project's README, description, or source code
 
 Unmaintained projects
 ~~~~~~~~~~~~~~~~~~~~~
 
-A project is considered *unmaintained* when ALL of the following are met:
+A project is considered *unmaintained* when ANY of the following are met:
+
+* The configured VCS repository is unreadable. This can happen if the repository is deleted, credentials are broken or the git host permanently unresponsive.
+* The project is only serving content from releases and commits >6 months older than its source project (this happens with unmaintained forks).
+* All builds have failed for >6 months.
 
-* The project is only serving content from releases and commits of a project over 6 months old
 
 Reachability
 ~~~~~~~~~~~~
 
-for matters concerning projects that the user owns.
-In every case where contacting the user is necessary,
-the core team will reach out using one of the following means of contact:
+In every case where contacting the user is necessary, the core team will reach out using one of the following means of contact:
 
 * E-mail address on file in the user's profile
 * E-mail address on the GitHub user account where the project is pointed
 
-The core team will stop trying to reach the user after 2 weeks and the user
-will be considered *unreachable*.
+The core team will stop trying to reach the user after 3 weeks and the user will be considered *unreachable*.
+
+
+Implementation
+--------------
+
 
 Delisting of an abandoned project
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Projects that are determined to be *Unofficial* and *Unmaintained* will have a ``robots.txt`` file added that removes them from all search results.
+Projects that are determined to be unmaintained_ and unofficial_ will have a ``robots.txt`` file added that removes them from all search results:
+
+.. code-block:: text
+
+  # robots.txt
+  User-agent: *
+  Disallow: / # This project is delisted according to the Policy for Unofficial and Unmaintanied Projects
+
 
 Projects will be delisted when ALL of the following:
 
-* The project has been determined *unmaintained* and *unofficial* by the rules described above
-* The candidate is able to demonstrate their own failed attempts to contact the
-  existing owner
-* The project owner is *unreachable* by the Read the Docs team
+* The person who submits the report of the unmaintained_ and unofficial_ project also demonstrates failed attempts to contact the existing owner.
+* The project has been determined unmaintained_ and unofficial_ by the rules described above.
+* The project owner is unreachable_ by the Read the Docs team.
 * The core team does not have any additional reservations.
+* The core team notifies the source project's maintainers about changes made
```

Comment:
What is `source project` here? This feels a bit odd to me. Are we just going from the repo linked on the RTD Project?

Reply:
I'm confused about this as well.

I think it references to what @agjohnson said in https://github.com/readthedocs/readthedocs.org/pull/9731#discussion_r1023171957. However, in that comment he proposes to "remove reachability" as a mandatory step. Otherwise, it doesn't make sense to me to "notifies the maintainers again" if we already weren't able to reach them

Reply:
I liked writing "source project" in several draft texts. I will add a suggestion for a definition. I think it's useful to be refer to the project which the "unofficial documentation" is documenting. It needs a good term to not confuse the reader.

Reply:
I'd agree, "source project" is mixing a few terms. If we're talking about project on RTD, I'll refer to that as "project", but if we are talking about something outside RTD, I'll try to avoid the word "project" and use something more accurate -- GitHub _repository_, PyPI _package_, etc. It is confusing to reference though, there isn't a great term that fits everything here.

Also, if it changes this discussion, I was referring to contacting the maintainer(s) of the unofficial project, and contacting them _after_ we made the change to their project -- I posted more here: https://github.com/readthedocs/readthedocs.org/pull/9731#discussion_r1025570160

Reply:
> However, in that comment he proposes to "remove reachability" as a mandatory step.

Reading the whole bullet list in review now, and to clarify: if we want a reactive policy, we don't want there to be a 2/3/6 week waiting period for contact. From the abandoned project policy, we almost never get a response at this step.

This policy differs from the abandoned policy in that when someone reaches out about an unofficial fork, those docs are actively causing confusion for users. So, in this case, a reactive policy could resolve the issue more immediately than the abandoned project policy (usually always waiting 6 weeks). Other outlets in the past have been attempted DMCA takedowns on these projects, which doesn't quite work with open source forked code.

But if we feel this is too aggressive, I don't have issues otherwise with requiring contact first. I don't feel it's important to contact official maintainers if they aren't the reporting party though.

Reply:
@agjohnson can you give a more precise list of definitions? I would like something that can match these two candidates:

* Reported documentation project: In this policy, a project is reported as unofficial and unmaintained. We simply refer to this as "the project".
* Source project: The _reported documentation project_ documents another project, typically a software project. In most cases, this software project already has another documentation project elsewhere. The source project is the main subject of documentation.

Reply:
"Source project" is probably clearer as "official repository" or "official project" if there are no other terms available. Either way, we can likely skip mention here, as we're only planning on reaching out to the maintainers of the unofficial RTD project.

Reply:
Yea, I'm not really sure why we're talking about the official project at all? Notifying them seems like it would just be spam.

Reply:
The procedure to notify "official" or "upstream" project has been removed. It remains at the discretion of the ones implementing the delisting :+1: